### DISCUSSION TITLE
[SOLVED] Visibility option in the project

### DISCUSSION THREAD

**Discussion by akaderfidesol** [kimai/kimai#3345] [General]
We have a number of projects.

Each project has an assigned team.

Each team has a supervisor and several members.

The problem is that all the members of all the teams see all the projects, and we don't want that, we want to hide the projects they don't belong to, and only show the projects assigned to their team.

When we uncheck the visibility option in the projects, they disappear for everyone, and if they are made visible in the filters, all members see them again.

There is also the problem when it comes to recording the hours. If we don't make it visible, not even the members of that team that belongs to the project can see it, and if we make it visible, everyone can see all the projects.

In the administrator roles section we have not seen where to change these options.

Any solution?
Thank you!

**Comment by kevinpapst** [comment]
You have either set "view_all_data" permission for ROLE_USER or you have configured Teams wrong.
A project that is assigned to a team is only visible to the members of that team.

**Reply by akaderfidesol** [reply]
Hello @kevinpapst ,

Thanks for your fast answer.

We have the role options like this.
![My Remote Image](https://i.gyazo.com/fbea2771472d290a1d024ecb840242c7.png)
![My Remote Image](https://i.gyazo.com/6bccb8f70b109ca7291a31b2c00e3226.png)

If the project is not visible, it does not allow us to assign the project to the team, but if we check the visibility option it becomes visible to everyone.

Thanks agais!

**Reply by kevinpapst** [reply]
Forget the visible checkbox, this is a global setting that will hide the project everywhere. As described here:
https://www.kimai.org/documentation/project.html#visibility

**Reply by akaderfidesol** [reply]
SOLVED!

Thanks a lot!
